Python/Product/Analysis/Values/ClassInfo.cs

```diff
@@ -423,7 +423,9 @@ internal class ClassInfo : AnalysisValue, IReferenceableContainer, IHasRichDescr
             var result = new Dictionary<string, IAnalysisSet>(Scope.VariableCount);
 
             foreach (var v in Scope.AllVariables) {
-                v.Value.ClearOldValues();
+                if ((options & GetMemberOptions.ForEval) != GetMemberOptions.ForEval) {
```

Comment:
We have extension methods for these checks already. (In particular, it's much easier to read `!options.ForEval()` than the mask and `!=` comparison and figure out whether the flag should be set or unset.)

Reply:
OK, will update next PR. Didn't find those for `GetMembers`